### ISSUE TITLE
Issue with buster.c

### ISSUE THREAD

**Issue by joaocarlos** [rubund/graywolf#8]
Hi,

I am trying to compile this source in a OSX system. However I am getting the error bellow.

```
/graywolf/src/Ylib/buster.c:230:2: error: 
      non-void function 'Ybuster_addpt' should return a value [-Wreturn-type]
        return ;
```

Any ideas on what should I do? I know it is not usual to have such tools on OSX systems, but since it is a Unix basis, I can have some tools such YoSys installed.

**Comment by T045T** [comment]
I just spent a few hours trying to get graywolf to build on OSX. 

I'm starting to think that cleaning up the code would be easier than wrangling all the compiler options you'd need to...

**Comment by yageek** [comment]
You can lower the number of errors by adding `ADD_DEFINITIONS("-Wno-return-type")` in the main `CMakeLists.txt`

**Comment by madankumar92** [comment]
@yageek Adding this line doesn't help, it still gives non-void function error

**Comment by yageek** [comment]
May be adding this instead `ADD_DEFINITIONS("-std=c89 -Wno-return-type")`?

**Comment by StefanBruens** [comment]
Likely fixed in commit 6100c0ed3ab70248f57a21f4047d2fd3f246fc00

**Comment by rubund** [comment]
yes. all those should be fixed now
